auth-web/src/views/auth/HomeView.vue

```diff
@@ -271,8 +284,11 @@ export default class HomeView extends Vue {
     return !!this.currentAccountSettings
   }
 
-  private goToManageBusinesses (): void {
-    this.$router.push(`/${Pages.MAIN}/${this.currentAccountSettings.id}`)
+  private goToManageBusinesses (isNcRequest: boolean = false): void {
```

Comment:
Will be really helpful to use the full form of Nc.. I hope its something like NameChange.. will be easier to comprehend if the variable is named like isNameChangeRequest even if its a bit longer...just a minor change

Reply:
Absolutely, 100%. Will update this to make it more clear :)